README.md

```diff
@@ -1,11 +1,17 @@
-# H5Web, a web-based HDF5 file viewer
+# H5Web: React components for data visualization and exploration
 
 [![Demo](https://img.shields.io/website?down_message=offline&label=demo&up_message=online&url=https%3A%2F%2Fh5web.panosc.eu%2F)](https://h5web.panosc.eu/)
 
-H5Web is a stand-alone, web-based viewer to explore HDF5 files. It is built with
-React and uses
-[react-three-fiber](https://github.com/react-spring/react-three-fiber) for
-visualizations.
+H5Web is a collection of React components to visualize and explore data. It
+consists of two packages:
```

Comment:
Awesome, great wording :+1: 

Maybe you could just reword the sentence "H5Web's top-level `App` component" under the @h5web/app section below, to mention the word "viewer" there again?